plugins/chat/app/serializers/chat/direct_message_serializer.rb

```diff
@@ -2,15 +2,16 @@
 
 module Chat
   class DirectMessageSerializer < ApplicationSerializer
-    attributes :id
+    attribute :group
 
-    has_many :users, serializer: Chat::ChatableUserSerializer, embed: :objects
+    has_many :memberships, serializer: Chat::UserChannelMembershipSerializer, embed: :objects
 
-    def users
-      users = object.direct_message_users.map(&:user).map { |u| u || Chat::NullUser.new }
+    def group
+      object.chatable.group
+    end
 
-      return users - [scope.user] if users.count > 1
-      users
+    def memberships
+      object.user_chat_channel_memberships
     end
   end
 end
```

Comment:
We now check memberships and not users of a direct message channel

Reply:
It's not a group in the sense of a discourse group, group is just an attribute saying a direct message channel is not a one to one direct message channel.

so here it means all the users who can talk in this direct message channel.

Reply:
You think being members of a DM doesn't make sense? I think it does, but can be convinced otherwise

Reply:
I actually changed my mind alone here, and will keep users